RECOVERY.md

```diff
@@ -19,77 +21,90 @@ For example, when using [electrum](https://electrum.org/):
 
 When swapping funds to a `lightning-kmp` wallet, the following steps are performed:
 
-- funds are sent to a swap-in address via a swap transaction
+- funds are sent to a swap-in address via a swap transaction. 
 - we wait for that transaction to have enough confirmations
 - then, if the fees don't exceed the user's liquidity policy, these funds are moved into a lightning channel
 
+We use musig2 to aggregate user keys (user being the wallet) and server keys (server being the LSP: the ACINQ node): swap-in addresses are standard p2tr addresses, and
+swap-in transactions to your wallet are indistinguishable from other p2tr transactions.
+
 The swap transaction's output can be spent using either:
 
-1. A signature from the user's wallet and a signature from the remote node
+1. An aggregated musig2 signature built from a partial signature from the user's wallet and a partial signature from the remote node
 2. A signature from the user's wallet after a refund delay
 
 Funds can be recovered using the second option and [Bitcoin Core](https://github.com/bitcoin/bitcoin).
-This process needs at least Bitcoin Core 25.0.
-
-This process will become simpler once popular on-chain wallets (such as [electrum](https://electrum.org/)) add supports for output script descriptors.
-
-### Extract master keys
+This process needs at least Bitcoin Core 26.0.
 
-We don't directly export your extended master private key for security reasons, so you will need to manually insert it in the descriptor.
-You can obtain your extended master private key in [electrum](https://electrum.org/). After restoring your seed, type `wallet.keystore.xprv` in the console to obtain your master `xprv`.
+This process will become simpler once popular on-chain wallets (such as [electrum](https://electrum.org/)) add support for output script descriptors.
 
 ### Create recovery wallet
 
-Create a wallet to recover your funds using the following command:
+#### Compute your refund master private key
 
-```sh
-bitcoin-cli createwallet recovery
-```
+For security reasons, we don't directly the refund master private key used for swap-ins, so you will need to manually insert it in the descriptor.
```

Comment:
```suggestion
For security reasons, we don't directly export the refund master private key used for swap-ins, so you will need to manually insert it in the descriptor.
```

Reply:
fixed in https://github.com/ACINQ/lightning-kmp/pull/563/commits/5e18d0b06f2f6439f8ba2d3c52398f340bcff569